src/crimson/os/seastore/onode.h

```diff
@@ -35,6 +35,8 @@ struct onode_layout_t {
   omap_root_le_t xattr_root;
 
   object_data_le_t object_data;
+  local_object_id_le_t local_object_id{LOCAL_OBJECT_ID_NULL};
+  local_clone_id_le_t local_clone_id{LOCAL_CLONE_ID_NULL};
```

Comment:
Looks `local_object/clone_id` are duplicated to omap roots and object_data_t, if they contain valid laddrs.

Seems to me it is possible to identify `local_object/clone_id` at the onode creation time, and if this is correct, there is no need to implement them with extra deltas and delayed update mechanisms.

Also it is possible to save more bytes from `onode_layout_t` if omap roots and object_data_t must share the same clone-prefix.

Let me know what you think.

Reply:
Looks reasonable, I will drop these fields.